### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#149] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-09 172818](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/17a87e9a-173e-49f6-a731-c4bf38fdee9a)

---
![Screenshot 2023-11-09 172834](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/36cfa473-11e5-4577-8cc2-63e200b781cf)

**Comment by IonutCicio** [comment]
# Automorfismi interni

## Verificare che $\gamma_x$ è un automorfismo di $G$

### $\gamma_x$ è un omomorfismo

Infatti $\forall g, h \in G, \gamma_x(gh) = xghx^{-1} = xgx^{-1}xhx^{-1} = \gamma_x(g)\gamma_x(h)$

### $\gamma_x$ è biettiva

Dato che l'insieme di partenza e di arrivo è lo stesso, basta dimostrare che $\gamma_x$ è iniettiva, quindi $\ker(\gamma_x) = \set{1}$

Supponiamo per assurdo che $\exists g \in G \mid g \neq 1 \land \gamma_x(g) = 1$. Si ha $\gamma_x(g) = xgx^{-1} = 1 \implies xg = x \implies g = x^{-1}x \implies g = 1$ in contraddizione con l'ipotesi che $g \neq 1$ 

## Verificare che $\phi : G \to Aut(G) : x \to \gamma_x$ è un omomorfismo

$\forall x, y \in G \phi(xy) = \gamma_{xy}$, ma $\gamma_{xy}(g) = xyg(xy)^{-1} = xygy^{-1}x^{-1} = x\gamma_y(g)x^{-1} = \gamma_x(\gamma_y(g)) = \gamma_x \circ \gamma_y = \phi(x) \phi(y)$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/fa81af4d-ce5d-4eb3-b984-2fbd8da90324)
